src/servers/cron.ts

```diff
@@ -43,12 +43,19 @@ const main = async () => {
     if (result instanceof Error) throw result
   }
 
+  const deleteLndPaymentsBefore2Weeks = async () => {
+    const timestamp2Weeks = new Date(Date.now() - TWO_WEEKS_IN_MS)
```

Comment:
I think is better to use 1 or 2 months. 2 weeks is too short to handle current support cases

Reply:
I'll vote for 2 months. I think that as long as we only keep the payments data in a bounded time window, we won't let bolt.db grown to an unsustainable size.